src/platform/unix.rs

```diff
@@ -66,6 +88,7 @@ impl<T> EventLoopWindowTargetExtUnix for EventLoopWindowTarget<T> {
     #[inline]
     fn wayland_display(&self) -> Option<*mut raw::c_void> {
```

Comment:
For the sake of consistency, I think backend-specific methods should always be marked with `#[cfg(...)]`; not just when the method signature contains a backend-specific type.